### PR TITLE
PR: modification of article card component's style and code formatter

### PR DESCRIPTION
**Pull request by creative2113**
Hello, there!
I am a creativeMan.
Please check and merge code.
Best regards.

### CODE REVIEW

**Review by shinevue — APPROVED** [review]
Please solve conflict

**Review by techietrend — APPROVED** [review]
Good job.
Go ahead.

File: client/src/features/dashboard/components/AmountStats.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
```
Comment (BlackGhost2693) [inline client/src/features/dashboard/components/AmountStats.js:1]: remove its whitespace

**Review by joellake992 — APPROVED** [review]
Wonderful

**Review by BlackGhost2693 — APPROVED** [review]
Excellent

File: client/src/features/dashboard/components/AmountStats.js (hunk 1 above)
Comment (creative2113) [reply client/src/features/dashboard/components/AmountStats.js:1]: I see.
just a min.

**Review by creative2113 — COMMENTED** [review]
thanks friends.

File: client/src/features/dashboard/components/AmountStats.js (hunk 1 above)
Comment (joellake992) [reply client/src/features/dashboard/components/AmountStats.js:1]: No need

Comment (creative2113) [reply client/src/features/dashboard/components/AmountStats.js:1]: I will update again.

**Review by gitMan-stack — COMMENTED** [review]
I think this is changed wonderfully

File: client/src/features/dashboard/components/AmountStats.js (hunk 1 above)
Comment (creative2113) [reply client/src/features/dashboard/components/AmountStats.js:1]: just done.

**Review by gitMan-stack — APPROVED** [review]
corrected wonderfully

**Comment by creative2113** [comment]
thanks for @gitMan-stack .

**Review by shinevue — APPROVED** [review]
good job

**Pull request merged**